website/docs/integrator-section/webapp-tutorial/deploy-document-store-webapp.md

```diff
@@ -14,17 +14,21 @@ Currently, there exists 2 web application:
 ## Prerequisites
 
 - Google Chrome web browser
-- Metamask setup
+- Metamask setup (You need to install Metamask extension in Google Chrome)
```

Comment:
You need to install **the** Metamask extension in Google Chrome?

Reply:
In oct-week3 branch, this line is updated to:
"- MetaMask extension in Google Chrome"